client/my-sites/checkout/composite-checkout/hooks/use-cached-domain-contact-details.ts

```diff
@@ -57,8 +52,7 @@ function useCachedContactDetailsForCheckoutForm(
 			? getCountryPostalCodeSupport( countriesList, cachedContactDetails.countryCode )
 			: false;
 
-	const checkoutStoreActions: CheckoutStoreMappedActions | undefined =
-		useDispatch( 'wpcom-checkout' );
+	const checkoutStoreActions = useDispatch( CHECKOUT_STORE );
```

Comment:
Noting that this intentionally introduces some type specificity regression, which will be resolved in the future when we upgrade to a newer version of `@wordpress/data`. We've explained that here: https://github.com/Automattic/wp-calypso/issues/74399#issuecomment-1469812339